spec/models/conversation_spec.rb

```diff
@@ -5,101 +5,97 @@
-      expect(cnv.ordered_participants.first).to eq(@user2.person)
-      expect(cnv.ordered_participants.last).to eq(@user1.person)
+      message_last
+      expect(conversation.ordered_participants.first).to eq(user2.person)
+      expect(conversation.ordered_participants.last).to eq(user1.person)
     end
   end
 
   describe '#first_unread_message' do
     before do
-      @cnv = Conversation.create(@create_hash)
-      @message = Message.create(:author => @user2.person, :created_at => Time.now + 100, :text => "last", :conversation_id => @cnv.id)
-      @message.increase_unread(@user1)
+      message_last.increase_unread(user1)
     end
 
     it 'returns the first unread message if there are unread messages in a conversation' do
-      @cnv.first_unread_message(@user1) == @message
+      conversation.first_unread_message(user1) == message_last
     end
 
     it 'returns nil if there are no unread messages in a conversation' do
-      @cnv.conversation_visibilities.where(:person_id => @user1.person.id).first.tap { |cv| cv.unread = 0 }.save
-      expect(@cnv.first_unread_message(@user1)).to be_nil
+      conversation.conversation_visibilities.where(person_id: user1.person.id).first.tap { |cv| cv.unread = 0 }.save
+      expect(conversation.first_unread_message(user1)).to be_nil
     end
   end
 
   describe '#set_read' do
+
     before do
-      @cnv = Conversation.create(@create_hash)
-      Message.create(:author => @user2.person, :created_at => Time.now + 100, :text => "first", :conversation_id => @cnv.id)
-             .increase_unread(@user1)
-      Message.create(:author => @user2.person, :created_at => Time.now + 200, :text => "last", :conversation_id => @cnv.id)
-             .increase_unread(@user1)
+      conversation
+      message_first.increase_unread(user1)
+      message_last.increase_unread(user1)
     end
 
     it 'sets the unread counter to 0' do
-      expect(@cnv.conversation_visibilities.where(:person_id => @user1.person.id).first.unread).to eq(2)
-      @cnv.set_read(@user1)
-      expect(@cnv.conversation_visibilities.where(:person_id => @user1.person.id).first.unread).to eq(0)
+      expect(conversation.conversation_visibilities.where(person_id: user1.person.id).first.unread).to eq(2)
+      conversation.set_read(user1)
+      expect(conversation.conversation_visibilities.where(person_id: user1.person.id).first.unread).to eq(0)
     end
   end
 
   context 'transport' do
+    let(:conversation_message) { conversation.messages.first }
+    let(:xml)                  { conversation.to_diaspora_xml }
+
     before do
-      @cnv = Conversation.create(@create_hash)
-      @message = @cnv.messages.first
-      @xml = @cnv.to_diaspora_xml
+      conversation
     end
 
     describe 'serialization' do
       it 'serializes the message' do
-        expect(@xml.gsub(/\s/, '')).to include(@message.to_xml.to_s.gsub(/\s/, ''))
+        expect(xml.gsub(/\s/, '')).to include(conversation_message.to_xml.to_s.gsub(/\s/, ''))
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.